handler/src/main/java/io/netty/handler/ssl/SniHandler.java

```diff
@@ -57,7 +57,7 @@
             InternalLoggerFactory.getInstance(SniHandler.class);
     private static final Selection EMPTY_SELECTION = new Selection(null, null);
 
-    private final AsyncMapping<String, SslContext> mapping;
+    protected final AsyncMapping<String, SslContext> mapping;
```

Comment:
does this need to be `protected`? can we just force users to use `super.lookup`?